setup.py

```diff
@@ -9,8 +9,10 @@
                        open("CHANGES.rst").read(),
       classifiers=[
         "Framework :: Plone",
+        "Framework :: Plone :: 5.0",
```

Comment:
plone.app.collection is the archetypes collection package, thus targeting Plone 4.3, while still compatible with 5.0.